spyder/utils/tests/test_conda.py

```diff
@@ -21,6 +21,8 @@
     get_conda_root_prefix, get_list_conda_envs, get_list_conda_envs_cache)
 
 
+CONDA_MISSING = (len(get_list_conda_envs()) == 0)
```

Comment:
Please change this line to

```python
if not is_anaconda():
    pytest.skip()
```

and remove the `skipif` you added in each test below. That will allow us to skip all tests in this file.

Reply:
Ah, I didn't know the `pytest.skip()` function; that's nice!